docs/pages/get-started/faq.md

```diff
@@ -324,19 +325,39 @@ See the [XMTP brand guidelines](https://github.com/xmtp/brand) GitHub repo.
 
 ### How should I handle the XMTP onboarding flow in my app?
 
-In your app onboarding flow, enable your users to activate XMTP DMs. User access to DMs can help with app engagement and re-engagement.
+In your app onboarding flow, enable your users to activate XMTP messaging. User access to messaging can help with app engagement and re-engagement.
 
 For example, here is a prompt to activate XMTP DMs in the onboarding flow to [claim a Lens handle](https://claim.lens.xyz/):
 
 ![activatedmsscreen-width-500px](https://raw.githubusercontent.com/xmtp/docs-xmtp-org/main/docs/pages/img/activate-dms-screen.png)
 
-In your app onboarding flow, request user permission to display app-specific push notifications to reach users outside of an app session.
+In your app onboarding flow, also request user permission to display app-specific push notifications. Enabling push notifications can help with user engagement and retention by allowing your app to reach users outside of active sessions. 
+
+Push notifications are particularly valuable for messaging apps, as they help ensure users don't miss important communications and can respond promptly to new messages.
+
+### Is there a way to get a list of all current XMTP-enabled wallets?
```

Comment:
Don't have a strong opinion, but do we need to display this question so high up on the list?

Reply:
Thank you, @richardhuaaa - such a great callout. I've moved it to the end of the Build section. Just one more question down - but a good move nonetheless!